Assets/Scripts/Controllers/Objects/DoorController.cs

```diff
@@ -43,13 +44,25 @@ public class DoorController : MonoBehaviour {
 			AlreadyActivated = true;
 			_isTransporting = true;
 
-			GameManager.Instance.LoadScene(
-				TransportTo switch {
-					TransportDestination.House => "houses",
-					TransportDestination.Special => "boss_arenas",
-					_ => "level_0"
-				}
-			);
+			Debug.Log("dd, YAEH");
```

Comment:
dd indeed.
There seems to be a bug or visual glitch when exiting a house whilst holding one of the movement keys.
The wipe seems to do a double take or smth.

Reply:
This is, however, not my issue, so I suppose just removing the debug log and the PR is all good-to-go?

Reply:
I'd argue that is is in fact your issue. While it is not necessarily directly caused by the screenwipe code, it is caused by the implementation of it in the doorcontroller and will need to be fixed anyways so why not now?

Got annoyed and did it myself this time.